src/main/resources/application.conf

```
@@ -37,6 +37,7 @@ agreement-process {
     user-registry                  = ${USER_REGISTRY_URL}
   }
 
+  events-queue-name         = ${EVENTS_QUEUE_NAME}
```

```suggestion
  archiving-purposes-queue-name         = ${ARCHIVING_PURPOSES_QUEUE_NAME}
```
Probably this service will push the same event twice into another queue (I'm talking about the eservice version archiving feature).
So may I suggest renaming like above?